.gitlab/functional_test/security_agent.yml

```
@@ -15,7 +15,7 @@
   retry: 0
   variables:
     AGENT_MAJOR_VERSION: 7
-    DD_PIPELINE_ID: $CI_PIPELINE_ID-a7
+    DD_PIPELINE_ID: $CI_PIPELINE_ID-functests-a7
```

This `DD_PIPELINE_ID` variable is part of the resource group name on Azure resources. This change is causing issues in some test cases, eg.:

```Failed to complete #create action: ['resource_group_name' should satisfy the constraint - 'MaxLength': '90'] on system-probe-test-ubuntu-16-04-44-azure-x86-64```

as the resource group name in that case is `kitchen-system-probe-test-ubuntu-16-04-44-azure-x86-64-20230112T144958pl12412126-functests-a7`, which is 93 characters long.

I'd suggest:
```suggestion
    DD_PIPELINE_ID: $CI_PIPELINE_ID-fnct
```

or some other kind of shorthand to refer to these tests (you can also remove the `-a7` part because you don't need it, you don't have separate Agent 6 / Agent 7 tests).

---

Ah that's right, let me do this change. Thanks !